### PR TITLE
[Inspiration] Video player design refresh

### PR DESCRIPTION
**Pull request by jameswklassen**
### ⚠️ **Depends on**
- [x] #35 (adding playback speed controls into d2l-video)
- [x] Brightspace/d2l-seek-bar#21 (change default seek bar styling/overrides)

## Reasoning
Refresh the design of `d2l-video` to scale better to smaller displays 
- Move the seek bar above the information
- Autohide controls when video is playing and mouse isn't moving

## Screenshots 
d2l-video mockups

### 250px (before/after)
250px
Screen Shot 2020-06-29 at 10 05 21 AM

### 350px (before/after)
Screen Shot 2020-06-17 at 12 58 03 PM
Screen Shot 2020-06-29 at 10 08 54 AM

### CODE REVIEW

File: d2l-video.js [hunk 1]
```diff
@@ -358,6 +384,29 @@ Polymer({
 		'esc': '_closeControls'
 	},
 
+	mousemove: function() {
+		this._showControlsTimeout();
+	},
+
+	_showControlsTimeout: function() {
```
Comment (jstefaniuk-d2l) [inline d2l-video.js:391]: It may be good if this could be renamed to something not exposing it's implementation

File: d2l-video.js [hunk 2]
```diff
@@ -248,17 +269,17 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-video">
...
 						</div>
 					</template>
 				</div>
 				<div class="layout horizontal center flex time-container" dir="ltr">
 					<div class="time-control time-control-left d2l-body-compact">{{ _formatTime(currentTime) }}</div>
-					<div class="flex seek-control">
-						<d2l-seek-bar id="seekBar" value="[[ percentComplete ]]" immediate-value="{{ immediateValue }}" aria-label$="[[localize('SeekBar')]]" on-drag-start="_onSeekStart" on-drag-end="_onSeekEnd" on-position-change="_onPositionChange"></d2l-seek-bar>
-					</div>
+					<div class="time-control-separator d2l-body-compact">/</div>
+					<!-- <div class="flex seek-control">
```
Comment (jstefaniuk-d2l) [inline d2l-video.js:281]: leftover commented out code

File: d2l-video.js [hunk 3]
```diff
@@ -331,6 +352,18 @@ Polymer({
 				2,
 			],
 		},
+		fadeInBuffer: {
+			type: Boolean,
+			value: false
+		},
+		timer: {
```
Comment (jstefaniuk-d2l) [inline d2l-video.js:359]: You only need to define properties like this if you:
- need to accept these properties as parameters
- need to bind the values in a template

So this and https://github.com/Brightspace/d2l-video/pull/37/files#diff-838116bb0060d5c8e25dcfd316361bc3R355 shouldn't be needed

File: d2l-video.js [hunk 4]
```diff
@@ -331,6 +352,18 @@ Polymer({
...
+			type: Object,
+			value: null
+		},
+		controlsVisible: {
```
Comment (jstefaniuk-d2l) [inline d2l-video.js:363]: should this be prepended with `_`?

**Comment by jameswklassen** [comment]
@ctwomey1 updated README screenshot

**Pull request merged**